### PR TITLE
[ja] Translate tasks/administer-cluster/use-cascading-deletion/ into Japanese

### PR DESCRIPTION
**Pull request by s-kawamura-w664**
Translated docs/tasks/administer-cluster/use-cascading-deletion.md into Japanese.

Fixes: #39958

### CODE REVIEW

File: content/ja/docs/tasks/administer-cluster/use-cascading-deletion.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+       "deletionTimestamp": "2021-07-08T20:27:39Z",
+       "finalizers": [
+         "foregroundDeletion"
+       ]
+       ...
+   ```
+
+## バッググラウンドカスケード削除を使用する {#use-background-cascading-deletion}
+
+1. [サンプルのDeploymentを作成する](/ja/docs/tasks/run-application/run-stateless-application-deployment/#creating-and-exploring-an-nginx-deployment).
```
Comment (atoato88) [inline content/ja/docs/tasks/administer-cluster/use-cascading-deletion.md:93]:
```suggestion
1. [サンプルのDeploymentを作成する](/ja/docs/tasks/run-application/run-stateless-application-deployment/#creating-and-exploring-an-nginx-deployment)。
```
Very small nit...

**Review by atoato88 — COMMENTED** [review]
Almost LGTM!
I left a very small format comment...

File: content/ja/docs/tasks/administer-cluster/use-cascading-deletion.md (hunk 1 above)
Comment (s-kawamura-w664) [reply content/ja/docs/tasks/administer-cluster/use-cascading-deletion.md]: Modified.

**Comment by s-kawamura-w664** [comment]
@atoato88 
I fixed your suggestion.
Please confirm.

**Comment by atoato88** [comment]
/lgtm
:+1:

**Review by nasa9084 — COMMENTED** [review]
/approve

**Pull request merged**